prow/config/values.yaml

```diff
@@ -42,7 +42,7 @@ hook:
   scrapeMetrics: false
   image: gcr.io/k8s-prow/hook:v20221121-dfec9ce18d
   service:
-    type: 'LoadBalancer'
+    type: 'ClusterIP'
```

Comment:
Does the ingress automatically create a LoadBalancer to manage the `http.paths`?

Reply:
@RedbackThomson if we're changing this from an external load balancer to cluster-internal service management, does that mean we can get rid of the [aws-load-balancer-controller dependency](https://github.com/aws-controllers-k8s/test-infra/blob/40700938cb4cfa1dfb2fae81131349b0bad5a522/infra/lib/ci-cluster.ts#L62)?

Reply:
We are still using the ALB controller to create an ALB for the ingress (https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/ingress/annotations/), which filters by path down to each of the services.